CoreWidgetProvider/Helpers/DataManager.cs

```diff
@@ -20,7 +20,7 @@ public DataManager(DataType type, Action updateWidget)
         updateAction = updateWidget;
         dataType = type;
 
-        updateTimer = new Timer(OneSecondInMilliseconds);
+        updateTimer = new Timer(3 * OneSecondInMilliseconds); // Three seconds to update
```

Comment:
This means that data will only be collected every 3 seconds, so we lose 2/3 of the data points in the charts, and they look very different from the Task Manager charts. Can we still do most of the updating every second, but only  update the UI every 3?

But then, what happens if the widgets aren't all in sync -- will it look weird if some are updating on second 0, some on 1, some on 2? Can we only update the list part every 3 seconds instead? PM/Design may want to weight in.

Reply:
Makes sense, I can do this change then. Personally I don't mind they all no longer updating in sync, but I agree it can be really strange if we have they all open at the same time.

I can remove this change from this PR at the moment and we discuss that change later.